### PR TITLE
Improve `post_date_column_time` filter description

### PR DESCRIPTION
**Pull request by crstauf**
Add "scheduled" to filter description.

Trac ticket: https://core.trac.wordpress.org/ticket/59029

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/includes/class-wp-posts-list-table.php [hunk 1]
```diff
@@ -1228,7 +1228,7 @@ public function column_date( $post ) {
 		}
 
 		/**
-		 * Filters the published time of the post.
+		 * Filters the published/scheduled time of the post.
```
Comment (mukeshpanchal27) [inline src/wp-admin/includes/class-wp-posts-list-table.php]: Should we add `Unpublished` also? 🤔

Comment (crstauf) [reply src/wp-admin/includes/class-wp-posts-list-table.php]: What's an unpublished time?

Comment (mukeshpanchal27) [reply src/wp-admin/includes/class-wp-posts-list-table.php]: Check line no 1186.

**Review by audrasjb — APPROVED** [review]

**Review by mukeshpanchal27 — APPROVED** [review]

**Comment by audrasjb** [comment]
committed in https://core.trac.wordpress.org/changeset/56436

**Pull request closed without merging**